server/index-node/src/resolver.rs

```diff
@@ -286,6 +287,9 @@ where
         let mut response: BTreeMap<String, q::Value> = BTreeMap::new();
         response.insert("features".to_string(), features);
         response.insert("errors".to_string(), errors);
+        if let Some(network_name) = network {
+            response.insert("network".to_string(), q::Value::String(network_name));
```

Comment:
Maybe this is a picky comment but I think it would make more sense to move the `q::Value::String` construction to the `Ether::Left` match branch, instead of here, since `features` and `errors` are already created with `q::Value::List` there.